content/docs/esc/integrations/rotated-secrets/aws-iam.md

```diff
@@ -0,0 +1,128 @@
+---
+title: aws-iam
+title_tag: aws-iam Pulumi ESC Rotator
+meta_desc: The `aws-iam` rotator enables you to rotate access credentials for an AWS IAM User.
+h1: aws-iam
+menu:
+  esc:
+    identifier: aws-iam
+    parent: esc-rotated-secrets
+    weight: 1
+---
+
+The `aws-iam` rotator enables you to rotate access credentials for an AWS IAM user in your Environment. Check out the [aws-login documentation](/docs/esc/integrations/dynamic-login-credentials/aws-login/) to learn more about authenticating with AWS.
```

Comment:
Can we add a section on the minimum permissions required by the role used for rotation? 

https://github.com/pulumi/docs/pull/14106#discussion_r1960852737